File: DSA_basics/arrays/q1/solution.py
def largest(arr):

    if(len(arr) == 0):
        return -1

    large = arr[0]
    for i in range(0,len(arr)):
        if(arr[i] > large):
            large = arr[i]
    return large       

arr = [1,2,3,4,8,8]
print("The largest integer in the array is:",largest(arr))

File: DSA_basics/arrays/q1/solution.cpp

// 1. Find the largest element in an array

#include<iostream>
#include<vector>
using namespace std;

int largest(vector<int> arr){

    if(arr.size() == 0){
        return -1;
    }

    int large = arr[0];
    for(int i = 0; i < arr.size(); i++){
        if(arr[i] > large){
            large = arr[i];
        }
    }
    return large;
}
int main(){
    vector<int> arr = {};
    cout << "The largest element in the array is:" << largest(arr) << endl;

    return 0;
}